public/app/features/alerting/unified/RuleEditor.tsx

```diff
@@ -45,9 +46,7 @@ const ExistingRuleEditor: FC<ExistingRuleEditorProps> = ({ identifier }) => {
       <Page.Contents>
         <InfoBox severity="warning" title="Rule not found">
           <p>Sorry! This rule does not exist.</p>
-          <a href="/alerting/list">
-            <Button>To rule list</Button>
-          </a>
+          <LinkButton href={locationUtil.assureBaseUrl('/alerting/list')}>To rule list</LinkButton>
```

Comment:
do not think locationUtil.assureBaseUrl is needed

Reply:
removed